ssm_jax/lgssm/core.py

```diff
@@ -0,0 +1,274 @@
+import jax.numpy as jnp
+import jax.random as jr
+from jax import lax
+
+import chex
+
+from distrax import MultivariateNormalFullCovariance as MVN
+
+
+@chex.dataclass
+class LGSSMParams:
+    """Lightweight container for LGSSM parameters."""
+    initial_mean: chex.Array
+    initial_covariance: chex.Array
+    dynamics_matrix: chex.Array
+    dynamics_input_weights: chex.Array
+    dynamics_covariance: chex.Array
+    emission_matrix: chex.Array
+    emission_input_weights: chex.Array
+    emission_covariance: chex.Array
+    is_stationary: chex.Array = True
+
+    @property
+    def state_dim(self):
```

Comment:
This may not work in the non-stationary case...

Reply:
I think it will since the nonstationary emissions should be shape `(T, D_emission, D_latent)`